scripts/dowhop-reviews.js

```diff
@@ -110,11 +119,22 @@ function readCommentsFromDatabase(reviewType, commentDiv) {
   var reviewRef = database.ref().child('doWhopDescription/' + currentDoWhopKey + '/reviews');
   var commentRef = reviewRef.child(reviewType).child('/comments');
   commentRef.on('value', function(snapshot) {
-    var comments = snapshot.val();
     var div = document.createElement('div');
     commentDiv.innerHTML = '';
-    _.forEach(comments, function(comment) {
-      div.innerHTML += '<blockquote>' + comment + '</blockquote>';
+    var commentsDetails = snapshot.val();
+    _.forEach(commentsDetails, function(commentDetail) {
+      div.innerHTML +=
+        '<blockquote>' +
+        commentDetail.comment +
+        '</blockquote>' +
+        '<img class="comment-headshot-pic" src="' +
```

Comment:
At the moment, these don't appear to be working on the front-end. Is this expected?